Tests/Form/Widget/FormChoiceWidgetTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Form\Widget;
+
+use Symfony\Component\HttpKernel\Kernel;
+
+class FormChoiceWidgetTest extends BaseWidgetTest
+{
+    protected $type = 'form';
+
+    public function setUp()
+    {
+        parent::setUp();
+    }
+
+    /**
+     * For SF < 2.6, we use 'empty_data' to provide default empty value.
+     * For SF >= 2.6, we must use 'placeholder' to achieve the same.
+     */
+    public function testDefaultValueRendering()
+    {
+        //different options
+        if (version_compare(Kernel::VERSION, '2.6.0', '>=')) {
+            $params = array(
+                'placeholder' => 'Default value',
```

Comment:
Would use `Choose an option` instead of `Default value`, cause it is not a real value